Unreal/CMakeLists.txt

```
@@ -55,6 +55,12 @@ set (
   ${CARLA_UNREAL_ENGINE_PATH}/Engine/Binaries/${UE_SYSTEM_NAME}/UnrealEditor${EXE_EXT}
 )
 
+if (IS_DIRECTORY ${CARLA_UE_PATH}/Content/Carla)
```

```suggestion
if (IS_DIRECTORY ${CARLA_UE_PATH}/Content/Carla/Blueprints)
```

This will also check that `Content/Carla` is not empty